acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/signing/SecpSigningAcceptanceTest.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.tests.signing;
+
+import static io.restassured.RestAssured.given;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Collections.singletonMap;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.web3j.crypto.Sign.publicKeyFromPrivate;
+import static org.web3j.crypto.Sign.signedMessageToKey;
+import static tech.pegasys.signers.secp256k1.MultiKeyTomlFileUtil.createAzureTomlFileAt;
+import static tech.pegasys.signers.secp256k1.MultiKeyTomlFileUtil.createFileBasedTomlFileAt;
+import static tech.pegasys.signers.secp256k1.MultiKeyTomlFileUtil.createHashicorpTomlFileAt;
+
+import tech.pegasys.eth2signer.dsl.signer.SignerConfigurationBuilder;
+import tech.pegasys.eth2signer.tests.AcceptanceTestBase;
+import tech.pegasys.signers.hashicorp.dsl.HashicorpNode;
+import tech.pegasys.signers.secp256k1.HashicorpSigningParams;
+
+import java.io.File;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.SignatureException;
+
+import com.google.common.io.Resources;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.vertx.core.json.JsonObject;
+import org.apache.tuweni.bytes.Bytes;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariables;
+import org.junit.jupiter.api.io.TempDir;
+import org.web3j.crypto.Sign.SignatureData;
+
+public class SecpSigningAcceptanceTest extends AcceptanceTestBase {
+  private static final byte[] DATA_TO_SIGN = "42".getBytes(UTF_8);
+  private static final String PRIVATE_KEY =
+      "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63";
+
+  private static final String SIGN_ENDPOINT = "/signer/sign/{identifier}";
+
+  static final String FILENAME = "fe3b557e8fb62b89f4916b721be55ceb828dbd73";
+
+  @Test
+  @EnabledIfEnvironmentVariables({
+    @EnabledIfEnvironmentVariable(named = "AZURE_CLIENT_ID", matches = ".*"),
+    @EnabledIfEnvironmentVariable(named = "AZURE_CLIENT_SECRET", matches = ".*"),
+    @EnabledIfEnvironmentVariable(named = "AZURE_KEY_VAULT_NAME", matches = ".*")
+  })
+  @EnabledIfEnvironmentVariable(named = "AZURE_KEY_VAULT_NAME", matches = ".*")
```

Comment:
can probably make these more simple - i.e. we KNOW that the multitoml file loading stuff works (due to tests in Signer) - just need to prove we've wired the HTTP endpoint up to the signerprovider

Reply:
Could possibly test most of this in an integration from the http endpoint to the SignerProvider. But I like having this basic AT for these providers as it also shows that the Runner and our ArtifactSignerProvider adapters are working.